cli/config/configuration-fc.toml.in

```diff
@@ -137,7 +137,7 @@ block_device_driver = "@DEFBLOCKSTORAGEDRIVER_FC@"
 # that it is running on top a VMM. This will result in the runtime
 # behaving as it would when running on bare metal.
 # 
-#disable_nesting_checks = true
+disable_nesting_checks = true
```

Comment:
Looks like this flag was being used just for the ebabling/disabling legacy virtio. If so, we should probably get rid of this flag all together (will help in reducing our configuration bloat as well)
Wdyt @jodh-intel @grahamwhaley

Reply:
Sounds reasonable to me, but let's get input from @mcastelino who added this feature.